contrib/bobtoolz/StdAfx.h

```diff
@@ -35,7 +35,7 @@
 
 #include "time.h"
 
-#if defined( __linux__ ) || defined( __FreeBSD__ ) || defined( __APPLE__ )
+#if defined( __linux__ ) || defined( __FreeBSD__ ) || defined( __APPLE__ ) || defined( __OpenBSD__ ) || defined( __NetBSD__ )
```

Comment:
I'd rather we set a radiant-only define for this than carrying/changing this long line everywhere.

Reply:
Something like:
```c
#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__)
# define __BSD__
#endif
```
to shorten it?

Reply:
yeah a macro like __BSD__ will help... because adding/update all defines.. take a lot of time...